<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DirectorWindow</class>
 <widget class="QMainWindow" name="DirectorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>747</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="cred_dir_tab">
       <attribute name="title">
        <string>Персональные данные</string>
       </attribute>
       <widget class="QLabel" name="email_dir_value">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>220</y>
          <width>281</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="email_dir_label">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>220</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
       <widget class="QLabel" name="phone_dir_label">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>250</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Номер телефона</string>
        </property>
       </widget>
       <widget class="QLabel" name="phone_dir_value">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>250</y>
          <width>311</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="name_dir_value">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>180</y>
          <width>291</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="name_dir_label">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>180</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Имя</string>
        </property>
       </widget>
       <widget class="QPushButton" name="edit_cred_dir_button">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>380</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Редактировать данные</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="books_dir_tab">
       <attribute name="title">
        <string>Книги</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTableWidget" name="books_table_widget">
          <column>
           <property name="text">
            <string>Название</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Автор</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Жанр</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Количество</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Год публикации</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Цена</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Аннотация</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="opers_dir_tab">
       <attribute name="title">
        <string>Операторы</string>
       </attribute>
       <widget class="QTableWidget" name="all_operators_table">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>761</width>
          <height>291</height>
         </rect>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Имя</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Email</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Номер телефона</string>
         </property>
        </column>
       </widget>
       <widget class="QPushButton" name="reg_oper_button">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>350</y>
          <width>161</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Регистрация оператора</string>
        </property>
       </widget>
       <widget class="QPushButton" name="delete_operator_button">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>390</y>
          <width>161</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Деактивация оператора</string>
        </property>
       </widget>
       <widget class="QComboBox" name="operator_emails_cb">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>410</y>
          <width>131</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="save_delete_oper">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>390</y>
          <width>111</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Деактивировать</string>
        </property>
       </widget>
       <widget class="QPushButton" name="edit_data_button">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>430</y>
          <width>161</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Изменить данные</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="edit_name_value">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>450</y>
          <width>191</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="edit_name_label">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>430</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Имя</string>
        </property>
       </widget>
       <widget class="QLabel" name="edit_email_label">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>480</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="edit_email_value">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>500</y>
          <width>191</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="edit_phone_value">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>550</y>
          <width>191</width>
          <height>24</height>
         </rect>
        </property>
        <property name="maxLength">
         <number>11</number>
        </property>
       </widget>
       <widget class="QLabel" name="edit_phone_label">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>530</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Номер телефона</string>
        </property>
       </widget>
       <widget class="QPushButton" name="edit_save_button">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>450</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Сохранить</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="status_dir_tab">
       <attribute name="title">
        <string>Статусы</string>
       </attribute>
       <widget class="QTableWidget" name="all_statuses">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>731</width>
          <height>241</height>
         </rect>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Название</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Описание</string>
         </property>
        </column>
       </widget>
       <widget class="QPushButton" name="edit_status_button">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>290</y>
          <width>131</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Редактировать статус</string>
        </property>
       </widget>
       <widget class="QComboBox" name="statuses_cb">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>290</y>
          <width>121</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="status_diff_value">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>290</y>
          <width>171</width>
          <height>111</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="save_new_diff_status">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>330</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Сохранить</string>
        </property>
       </widget>
       <widget class="QPushButton" name="add_new_status_button">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>330</y>
          <width>131</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Добавить статус</string>
        </property>
       </widget>
       <widget class="QPushButton" name="save_new_status_button">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>330</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Сохранить</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="new_status_name_value">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>290</y>
          <width>121</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="status_name_label">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>270</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Название</string>
        </property>
       </widget>
       <widget class="QLabel" name="status_diff_label">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>270</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Описание</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="opders_dir_tab">
       <attribute name="title">
        <string>Заказы</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTableWidget" name="all_orders">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Номер заказа</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Имя</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Номер телефона</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Email</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Стоимость</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Дата оформления</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Статус</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QCheckBox" name="only_cancelled_cb">
             <property name="text">
              <string>Только отменённые</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="stats_dir_tab">
       <attribute name="title">
        <string>Статистика</string>
       </attribute>
       <widget class="QDateEdit" name="start_date">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>50</y>
          <width>110</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QDateEdit" name="end_date">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>50</y>
          <width>110</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="date_label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Временной промежуток</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>50</y>
          <width>141</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Показать статистику</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="most_popular_book_table">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>140</y>
          <width>711</width>
          <height>71</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>Название</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Автор</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Жанр</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Количество</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Год публикации</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Цена</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Аннотация</string>
         </property>
        </column>
       </widget>
       <widget class="QLabel" name="most_popular_book_label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>120</y>
          <width>161</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Самая популярная книга</string>
        </property>
       </widget>
       <widget class="QLabel" name="most_popular_author_label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>220</y>
          <width>161</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Самый популярный автор:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="num_of_clients_sb">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>80</y>
          <width>42</width>
          <height>25</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
       <widget class="QLabel" name="date_label_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>80</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Количество клиентов</string>
        </property>
       </widget>
       <widget class="QLabel" name="most_popular_genre_label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>310</y>
          <width>161</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Самый популярный жанр:</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="most_profit_clients_table">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>420</y>
          <width>711</width>
          <height>71</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>Имя</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Номер телефона</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Email</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Дата рождения</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Статус</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Сумма заказов</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string/>
         </property>
        </column>
       </widget>
       <widget class="QLabel" name="most_profit_clients_label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>400</y>
          <width>191</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Самые прибыльные клиенты</string>
        </property>
       </widget>
       <widget class="QLabel" name="money_value">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>510</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="money_label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>510</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Выручка:</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="most_popular_author_value">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>240</y>
          <width>711</width>
          <height>61</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>Имя</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Дата рождения</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Дата смерти</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Книг заказано</string>
         </property>
        </column>
       </widget>
       <widget class="QTableWidget" name="most_popular_genre_value">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>310</y>
          <width>211</width>
          <height>61</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>Название</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Всего заказано</string>
         </property>
        </column>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>805</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
